lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -1467,19 +1475,23 @@ bool IsMacroDefinition(StringRef arg) {
 }
 
 bool ShouldUnique(StringRef arg) {
-  return IsMacroDefinition(arg);
+  for (auto &flag : args_to_unique)
+    if (arg.startswith(flag))
+      return true;
+  return false;
 }
 } // namespace
 
-void SwiftASTContext::AddExtraClangArgs(std::vector<std::string> ExtraArgs) {
-  swift::ClangImporterOptions &importer_options = GetClangImporterOptions();
+namespace lldb_private {
+void AddExtraClangArgs(const std::vector<std::string>& source,
```

Comment:
I've converted this to free function for easier unit testing

Reply:
nice!
I think a static function of SwiftASTContext would also work and wouldn't pollute the lldb_private namespace.

Reply:
No problem